src/main/java/com/example/Customer.java

```diff
@@ -0,0 +1,23 @@
+package com.example;
+
+/**
+ * Public class Customer with the field loyaltyCard and
+ * constructor
```

Comment:
This documentation does not add any value. I can see that from the code. Do Not Repeat Yourself (DRY).

You need to describe the class from a modeling perspective. Something like: “A customer is a person who visits a shop for buying goods.” Then, add something about possession of a loyalty card.